packages/ripple-binary-codec/test/definitions.test.js

```diff
@@ -1,5 +1,6 @@
 const { encode, decode, XrplDefinitions } = require('../src')
 const normalDefinitionsJson = require('../src/enums/definitions.json')
+const altDefinitions = require('xrpl-binary-codec-prerelease/dist/enums/definitions-xahau.json')
```

Comment:
I'd recommend having the definitions file be just a copy-pasted test file in the `fixtures` folder, since it doesn't need to be kept up-to-date for the tests to still work.

Reply:
You know what: I propose that you decide internally how you want to resolve this :) Lmk if there's anything I can do at this point.

Reply:
I agree with @mvadari approach.